schedule.go

```diff
@@ -8,13 +8,19 @@ type (
 	Schedule struct {
 		Minute, Hour, Dom, Month, Dow bitset
 
-		// TODO: support the timezone option
-		// Location *time.Location
+		Location *time.Location
 	}
 )
 
+// Next returns the next time matched with the expression.
 func (s *Schedule) Next(t time.Time) time.Time {
 	loc := time.UTC
+	if s.Location != nil {
+		loc = s.Location
+	}
+
+	origLoc := t.Location()
+	t.In(loc)
```

Comment:
Change the location of `t` when it evaluates the next time, and revert the location when it returns.